setup.cfg

```
@@ -1,10 +1,89 @@
+[metadata]
+name = python-lsp-server
+author = Python Language Server Contributors
+description = Python Language Server for the Language Server Protocol
+url = https://github.com/python-lsp/python-lsp-server
+long_description = file: README.md
+long_description_content_type = text/markdown
+
+[options]
+packages = find:
+python_requires = >=3.7
+install_requires =
+    jedi>=0.17.2,<0.19.0
+    python-lsp-jsonrpc>=1.0.0
+    pluggy
+    ujson>=3.0.0
+    setuptools>=39.0.0
+setup_requires = setuptools>=44; wheel; setuptools_scm[toml]>=3.4.3
+
+[options.packages.find]
+exclude = contrib; docs; test; test.*; test.plugins; test.plugins.*
+
+[options.extras_require]
+all =
+    autopep8>=1.6.0,<1.7.0
+    flake8>=4.0.0,<4.1.0
+    mccabe>=0.6.0,<0.7.0
+    pycodestyle>=2.8.0,<2.9.0
+    pydocstyle>=2.0.0
+    pyflakes>=2.4.0,<2.5.0
+    pylint>=2.5.0
+    rope>=0.10.5
+    yapf
+autopep8 = autopep8>=1.6.0,<1.7.0
+flake8 = flake8>=4.0.0,<4.1.0
+mccabe = mccabe>=0.6.0,<0.7.0
+pycodestyle = pycodestyle>=2.8.0,<2.9.0
+pydocstyle = pydocstyle>=2.0.0
+pyflakes = pyflakes>=2.4.0,<2.5.0
+pylint = pylint>=2.5.0,<2.10.0
```

```suggestion
pylint = pylint>=2.5.0
```

There's no upper constraint for Pylint now (there was at some point though).